src/tests/test_package_loading.py

```diff
@@ -136,7 +136,7 @@ def test_invalid_package_name(selenium):
 )
 def test_load_packages_multiple(selenium_standalone, packages):
     selenium = selenium_standalone
-    selenium.load_package(packages)
+    loaded_packages = selenium.load_package(packages)
```

Comment:
[selnium.load_package](https://github.com/pyodide/pytest-pyodide/blob/045c569f8583f2bb456045a43ac172c7e74889ce/pytest_pyodide/runner.py#L328-L329) is a wrapper function for `pyodide.loadPackage`, but it does not return the return value.

Maybe we can update `pytest-pyodide`, but it would need a few steps, so for now let's call pyodide.loadPackage directly. You can do:

```
selenium.run_js("return await pyodide.loadPackage('...')")
```

Reply:
I have fixed the tests with your suggestion, but it looks like it is failing only on safari. Could you perhaps take a look and tell what may be wrong here? @ryanking13